[2024-09-22T01:11:41.036+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_eval.model_cross_validation.RandomForestRegressor manual__2024-09-22T01:11:34.547958+00:00 [queued]>
[2024-09-22T01:11:41.073+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_eval.model_cross_validation.RandomForestRegressor manual__2024-09-22T01:11:34.547958+00:00 [queued]>
[2024-09-22T01:11:41.074+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2024-09-22T01:11:41.132+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): model_cross_validation.RandomForestRegressor> on 2024-09-22 01:11:34.547958+00:00
[2024-09-22T01:11:41.213+0000] {standard_task_runner.py:60} INFO - Started process 4981 to run task
[2024-09-22T01:11:41.222+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'dag_eval', 'model_cross_validation.RandomForestRegressor', 'manual__2024-09-22T01:11:34.547958+00:00', '--job-id', '88', '--raw', '--subdir', 'DAGS_FOLDER/main.py', '--cfg-path', '/tmp/tmp2qxak1ik']
[2024-09-22T01:11:41.224+0000] {standard_task_runner.py:88} INFO - Job 88: Subtask model_cross_validation.RandomForestRegressor
[2024-09-22T01:11:41.294+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:194: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  SQL_ALCHEMY_CONN = conf.get("database", "SQL_ALCHEMY_CONN")

[2024-09-22T01:11:41.391+0000] {task_command.py:423} INFO - Running <TaskInstance: dag_eval.model_cross_validation.RandomForestRegressor manual__2024-09-22T01:11:34.547958+00:00 [running]> on host cc45b92b61e2
[2024-09-22T01:11:41.742+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag_eval' AIRFLOW_CTX_TASK_ID='model_cross_validation.RandomForestRegressor' AIRFLOW_CTX_EXECUTION_DATE='2024-09-22T01:11:34.547958+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-09-22T01:11:34.547958+00:00'
[2024-09-22T01:11:42.842+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-09-22T01:11:42.860+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=dag_eval, task_id=model_cross_validation.RandomForestRegressor, execution_date=20240922T011134, start_date=20240922T011141, end_date=20240922T011142
[2024-09-22T01:11:42.922+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-09-22T01:11:42.965+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
